app/controllers/application_controller.rb

```diff
@@ -15,7 +15,7 @@ class ApplicationController < ActionController::Base
 
   # Imitate logged in user
   def current_user_dev
-    @current_user ||= User.new(:username => "admin123", :firstname => "Atila")
+    @current_user_dev ||= User.find_by_username('dev123')
```

Comment:
@eshadatta Keep this `@current_user` in case there are any unforeseen repercussions. For instance authlogic and devise look for the `current_user` method but there may be inner workings that look for the instance variable instead and this would cause problems.